### DISCUSSION TITLE
InternLM pretrain llama2-7b之后，怎么做推理

### DISCUSSION THREAD

**Discussion by wen020** [InternLM/InternLM#748] [General]
使用InternLM pretrain llama2-7b之后，怎么用训练之后的模型做推理？

**Comment by MCplayerFromPRC** [comment]
> 使用InternLM pretrain llama2-7b之后，怎么用训练之后的模型做推理？

加载InternEvo格式权重并进行推理的最新版代码已经提交到[PR #260](https://github.com/InternLM/InternEvo/pull/260)
